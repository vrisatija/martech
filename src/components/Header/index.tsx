import React, { FC } from 'react';
import Images from '../../assets';

const Header: FC = () => (
  <div className="flex px-10 py-7 justify-between bg-blue-900 text-white absolute w-full">
    <div className="font-ibm-semi-bold">logo</div>
    <div className="flex items-center cursor-pointer">
      <span className="mr-2">Profile</span>
      <img src={Images.downArrow} className="w-3.5" alt="menu arrow" />
    </div>
  </div>
);

export default Header;
