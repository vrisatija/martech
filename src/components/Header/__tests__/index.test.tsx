import React from 'react';
import { render } from '@testing-library/react';
import Header from '..';

describe('Header Component', () => {
  it('renders header component', () => {
    const HeaderFragment = render(<Header />).asFragment();
    expect(HeaderFragment).toMatchSnapshot();
  });
});
