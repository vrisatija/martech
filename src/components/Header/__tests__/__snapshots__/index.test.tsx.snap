// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders header component 1`] = `
<DocumentFragment>
  <div
    class="flex px-10 py-7 justify-between bg-blue-900 text-white absolute w-full"
  >
    <div
      class="font-ibm-semi-bold"
    >
      logo
    </div>
    <div
      class="flex items-center cursor-pointer"
    >
      <span
        class="mr-2"
      >
        Profile
      </span>
      <img
        alt="menu arrow"
        class="w-3.5"
        src="downArrow.svg"
      />
    </div>
  </div>
</DocumentFragment>
`;
